oci_runtime.rst

```diff
@@ -31,168 +31,203 @@ tools from the OCI ecosystem.
-exec``.
-
-Future Development
-==================
-
-As newer Linux kernels and system software reach production environments, many
-of the limitations that required {Singularity} to operate quite differently from
-OCI runtimes are becoming less-applicable. From 3.11, {Singularity} development
-will focus strongly on greater OCI compliance for typical usage, while
-maintaining the same ease-of-use and application focus.
-
-You can read more about these plans in the following article and open community
-roadmap:
-
-* https://sylabs.io/2022/02/singularityce-4-0-and-beyond/
-* https://github.com/sylabs/singularityce-community
-
-.. _oci_mode:
+1. The :ref:`OCI Mode section <oci_mode>` of this page introduces the OCI
+   runtime (``--oci``), which runs OCI / Docker containers in their native
+   format.
+2. The :ref:`OCI Command Group section <oci_command>` of this page documents the
+   ``singularity oci`` commands, which provide a low-level means to run
+   {Singularity} SIF containers with a command line that matches other OCI
+   runtimes.
+3. The :ref:`Support for Docker <singularity-and-docker>` page discusses
+   limitations, compatibility options, and best practices for running OCI /
+   Docker containers with {Singularity}'s default runtime.
 
 ********************
 OCI Mode (``--oci``)
 ********************
 
-Overview
-========
-
-Beginning in {Singularity} 3.11, users can run an OCI / Docker container in its
-native format by adding the ``--oci`` flag to a ``run / shell /exec`` command:
+Users can run an OCI / Docker container in its native format by adding the
+``--oci`` flag to a ``run / shell /exec`` command:
 
 .. code::
 
-  $ singularity shell --oci docker://ubuntu 
+  $ singularity shell --oci docker://ubuntu
   2023/02/06 11:00:10  info unpack layer: sha256:677076032cca0a2362d25cf3660072e738d1b96fe860409a33ce901d695d7ee8
   Singularity> echo "Hello OCI World!"
   Hello OCI World!
 
 In ``--oci`` mode, the familiar ``singularity`` command line is used, and
-friendly defaults such as auto-mounting of the ``$HOME`` directory are still
-applied. The user experience is similar to when the ``--compat`` flag is used
-with the default runtime mode.
+friendly defaults familiar from the native {Singularity} runtime - such as
+auto-mounting of the ``$HOME`` directory - are still
+applied. By and large, the user experience is similar to using the ``--compat``
+flag with the native runtime, without the need to translate the OCI spec into an
+approximately equivalent {Singularity} definition file.
+
+.. note::
+
+  Like ``--compat`` mode in the native runtime, ``--oci`` mode provides a
+  writable container by default by using a tmpfs overlay.
+
+  See discussion of the ``--overlay`` flag, below, for other options, including
+  how to use persistent writable storage in OCI mode.
 
-The ``--oci`` mode only works with OCI containers, i.e. those from sources
+The ``--oci`` mode only works with OCI containers, i.e., those from sources
```

Comment:
Not true now we can run 'native SIF' in `--oci` mode. We can tackle this now, or this can just serve as a note for more edits later on.

Reply:
I'm happy to tackle this now, but I'm not entirely sure what the right approach is. We could remove this entire sentence (and just have this paragraph start with "SingularityCE retrieves and prepares ...". Or we could replace this lead-in sentence with something else... but I'm not sure what that should be.

Reply:
Let's take this on later. Better to tackle it later with the whole `--no-compat` to get traditional behaviour etc...  laying out the different ways you can run different things.